src/json_reporter.cc

```diff
@@ -78,7 +78,12 @@ bool JSONReporter::ReportContext(const Context& context) {
   out << indent << FormatKV("date", walltime_value) << ",\n";
 
   if (Context::executable_name) {
-    out << indent << FormatKV("executable", Context::executable_name) << ",\n";
+    // windows uses backslash for its path separator,
+    // which must be escaped in JSON otherwise it blows up conforming JSON
+    // decoders
+    std::string executable_name = Context::executable_name;
+    ReplaceAll(&executable_name, "\\", "\\\\");
+    out << indent << FormatKV("executable", executable_name) << ",\n";
```

Comment:
I'm not seeing this change reflected in tests.

Reply:
The tests do not explicitly check the full executable path, just the resulting endname. Properly escaped names show up in the appveyor Windows tests now.

Reply:
I changed it to check for either `//` or an escaped `\\\\` in the reporter output tests.